src/commands/run.rs

```diff
@@ -164,6 +164,12 @@ impl RunCommand {
         let engine = Engine::new(&config)?;
         let mut store = Store::new(&engine, Host::default());
 
+        // if consume fuel has been flagged, we want to instantiate the
+        // store with the set amount of fuel from the 'add_fuel' option
+        if self.common.consume_fuel {
+            store.add_fuel(self.common.add_fuel).unwrap();
```

Comment:
I don't think you want to panic here, even though this should never fail.

```suggestion
            store.add_fuel(self.common.add_fuel)?;
```